tools/archetype/binding.test/src/main/resources/archetype-resources/META-INF/MANIFEST.MF

```diff
@@ -7,22 +7,18 @@ Bundle-Version: ${version.replaceAll("-SNAPSHOT", ".qualifier")}
 Bundle-RequiredExecutionEnvironment: JavaSE-1.8
 Fragment-Host: ${package}
 Import-Package: ${package},
- ${package}.internal,
+ ${package}.handler,
  org.slf4j,
- groovy.json,
- groovy.lang,
- org.codehaus.groovy.reflection,
- org.codehaus.groovy.runtime,
- org.codehaus.groovy.runtime.callsite,
- org.codehaus.groovy.runtime.typehandling,
  org.eclipse.smarthome.core.events,
  org.eclipse.smarthome.core.thing.util,
- org.eclipse.smarthome.test,
  org.eclipse.smarthome.test.storage,
+ org.eclipse.smarthome.test.java,
  org.hamcrest;core=split,
+ org.mockito,
  org.junit;version="4.0.0",
  org.osgi.service.device,
  org.osgi.framework
+Require-Bundle: org.junit,org.mockito,org.hamcrest
```

Comment:
Is it only needed for hamcrest, or do the other two need to be included there as well?

Maybe we should update the guidelines of both projects as well because our archetype does not follow the guidelines anymore. And in my experience you should not in general enforce lower standards on your tests, that way it will (slowly) become a mess.